src/components/Profile/components/SchedulePanel/components/ScheduleCalendar/index.tsx

```diff
@@ -21,17 +22,21 @@ const GordonScheduleCalendar = ({ schedule, onSelectEvent }: Props) => {
 
   const courseFormat = schedule.courses.map((course) => {
     let title;
+
     if (course.location.includes('ASY')) {
-      title = `${course.title}${width >= 1500 ? ' | ' : ' '} ${course.location}`;
+      title = `${course.title.replaceAll(' ', '')} | ASYNC`;
+    } else if (course.location.includes('null')) {
+      title = `${course.title.replaceAll(' ', '')}`;
     } else {
-      title = `${course.title}${width >= 2212 ? ' | ' : ' '} ${course.location}`;
+      title = `${course.title.replaceAll(' ', '')} \n${course.location}`;
```

Comment:
I think this can most elegantly be handled with a switch case. 
https://www.w3schools.com/js/js_switch.asp
This will also allow for trickle down processes like ``${course.title.replaceAll(' ', '')}`` that is being used in both your else if and else statements

Reply:
We talked yesterday in class and decided it would be best done using ternary operators. The error we found has also been fixed by adding tempTitle instead of having title equal something.